src/integration/test-helpers.ts

```diff
@@ -27,20 +27,29 @@ export const randomString = (length) => {
   return result;
 };
 
-export const generateRecords = (
-  dimension: number,
-  quantity: number,
-  withSparseValues?: boolean,
-  withMetadata?: boolean
-): PineconeRecord[] => {
+export const generateRecords = ({
```

Comment:
Wanted to make this easier to work with, still feels a little clunky but using an object argument is helpful.